iam/policy/kms-action.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2015-2022 MinIO, Inc.
+//
+// This file is part of MinIO Object Storage stack
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package iampolicy
+
+// KMSAction - KMS policy action.
+type KMSAction string
+
+const (
+	// KMSCreateKeyAction - allow creating a new KMS master key
+	KMSCreateKeyAction = "kms:KMSCreateKey"
```

Comment:
we need to just keep 
```suggestion
	KMSCreateKeyAction = "kms:CreateKey"
```